clients/drcachesim/common/options.cpp

```diff
@@ -639,12 +652,14 @@ droption_t<bytesize_t> op_L0_filter_until_instrs(
     "full trace. Therefore TRACE_MARKER_TYPE_WINDOW_ID markers indicate start of "
     "filtered records.");
 
+// XXX i#7230: Currently the simulators count non-marker records here: we should
+// either change that to all records to match -skip_refs, or update these docs.
 droption_t<bytesize_t> op_warmup_refs(
-    DROPTION_SCOPE_FRONTEND, "warmup_refs", 0,
-    "Number of memory references to warm caches up",
-    "Specifies the number of memory references to warm up caches before simulation. "
-    "The warmup references come after the skipped references and before the "
-    "simulated references. This flag is incompatible with warmup_fraction.");
+    DROPTION_SCOPE_FRONTEND, "warmup_refs", 0, "Number of records to warm caches up",
```

Comment:
> Number of records to warm caches up

This should be updated too whenever we update the treatment of -warmup_refs to match -skip_refs. Currently it may be interpreted as "how many records to pass to the cache" which naturally does not include markers because markers are not passed to the cache.

Reply:
Added a comment.